src/modules/Elsa.Workflows.Management/Notifications/WorkflowDefinitionVersionsUpdate.cs

```diff
@@ -0,0 +1,6 @@
+namespace Elsa.Workflows.Management.Notifications;
+
+/// <summary>
+/// Represents an update to a specific workflow definition version.
+/// </summary>
+public record WorkflowDefinitionVersionsUpdate(string Id, string DefinitionId, bool UsableAsActivity);
```

Comment:
Given that this type represents an update to a single workflow definition version, the name should changed to `WorkflowDefinitionVersionUpdate`.